README.md

```diff
@@ -2,3 +2,26 @@
 
 This repository contains a collection of packages that can be used to develop
 against JupiterOne.
+
+## Integration Development
+
+Please reference the
+[development documentation](docs/integrations/development.md) for details about
+how to develop integrations with this SDK.
+
+## Integration SDK Development
+
+If you are making changes to the integration SDK and you want to test the
+changes in another integration project then it is recommended to automatically
+rebuild and link this project when changes are made.
+
+Steps to automatically build and link:
```

Comment:
There are lots of changes in these PRs, please forgive any oversight on my part... does this mean the `autobuild` script no longer works?

Reply:
Yeah, it's been removed in favor of doing `yarn build --watch`. I believe I document that in the next pr.